src/tools/sssctl/sssctl_logs.c

```diff
@@ -135,7 +135,7 @@ static const char *get_busname(TALLOC_CTX *mem_ctx, struct confdb_ctx *confdb,
             goto done;
         }
 
-        busname = sss_iface_domain_bus(mem_ctx, domain);
+        busname = talloc_strdup(mem_ctx, domain->conn_name);
```

Comment:
This should be in previous commit (`sbus: store dbus connection name in domain.conn_name`)

Reply:
Done.